src/index.ts

```diff
@@ -58,8 +59,9 @@ const renderWithCarbon = ({
 
 export default function supernova(env: Galaxy) {
   const {
-    flags: { isEnabled: isFlagEnabled },
+    flags: { isEnabled },
   } = env;
+  const areBasicFeaturesEnabled = isEnabled('PS_15585_SN_TABLE_BASIC_FEATURES');
```

Comment:
No real point of desctructuring this since is only used once. I would move this to inside `component` function.
```suggestion
const areBasicFeaturesEnabled = env.flags.isEnabled('PS_15585_SN_TABLE_BASIC_FEATURES');
```